search-app/resources/templates/search-docs-static.html

```diff
@@ -0,0 +1,36 @@
+{% extends "templates/structure.html" %}
+
+{% block title %}{{ site-title }} - {{ page-title }}{% endblock %}
+
+{% block top-nav %}
+<div class="eui-masthead-logo eui-application-logo">
+  <h1><a href="../../">CMR Search</a></h1>
+</div>
+<nav class="main-nav" role="navigation">
```

Comment:
It looks like we're repeating this header and footer in a couple of files. Do you think we could define header and footer templates and include them rather than duplicating?